x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/store/mock_activity_log.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EndpointAction } from '../../../../../common/endpoint/types';
+
+export const mockActivityLogResponse = (agentId: string): EndpointAction[] => {
+  const activityLog: EndpointAction[] = [
+    {
+      action_id: `${agentId}-action_id`,
+      '@timestamp': new Date().toJSON(),
```

Comment:
👏 did not know about the `.toJSON()` method of the Date class 👍

Reply:
now I redundant mock file 6a2cace04620e43ac4d3a4d57d52045fe5a5f7aa 😄